pom.xml

```diff
@@ -468,11 +469,6 @@ limitations under the License.
 
       <!-- JSch agentproxy -->
       <dependency>
-        <groupId>org.apache.servicemix.bundles</groupId>
-        <artifactId>org.apache.servicemix.bundles.jsch-agentproxy-jsch</artifactId>
-        <version>${jsch.agentproxy.bundle.version}</version>
```

Comment:
Nope. It is used [here](https://github.com/jclouds/jclouds-karaf/pull/77/files#diff-dd692a9b30ef09ae1bdc70c053148e11R36) to import the dependency to shade.